### ISSUE TITLE
Using the Recipe ID

### ISSUE THREAD

**Issue by PrakrutiM** [lizziehop/reading-cookbooks#2]
What do we do for the recipe ID if it is unclear which recipe a prefatory note is referring to? For instance, the prefatory note in mine refers to "French Dressing" but there are two recipes given for French Dressing in the chapter.

**Comment by PrakrutiM** [comment]
Non-recipe recipes - eg How to boil rice, how should they be tagged?

**Comment by noorka** [comment]
> Non-recipe recipes - eg How to boil rice, how should they be tagged?

These will be tagged as a recipe since that is how they act in the text. Create xml:id and tag ingredients as they come up in the text.

**Comment by reginahongcy** [comment]
When a recipe is telling you to use it for another recipe, how should it be tagged?  With div type="recipe_addition"?

I.e   [From a recipe for mustard dressing] Add the salt and pepper, and when the dressing is finished, use it for < div type="recipe_addition” corresp=“#pic22.14_celery_salad" >celery salad< /div >, fish, tomatoes, potatoes, etc.

**Comment by reginahongcy** [comment]
> When a recipe is telling you to use it for another recipe, how should it be tagged? With div type="recipe_addition"?
> 
> I.e [From a recipe for mustard dressing] Add the salt and pepper, and when the dressing is finished, use it for < div type="recipe_addition” corresp=“#pic22.14_celery_salad" >celery salad< /div >, fish, tomatoes, potatoes, etc.

Okay, I think I can answer my own question - the name tag should be used since it's a kind of enhancement or serving suggestion.
